sr_camera_launch/launch/ensenso.launch

```diff
@@ -0,0 +1,46 @@
+<launch>
+  <arg name="sim" default="true" />
+  <arg name="depth_registration" default="true" />
+  <arg name="camera" default="camera" />
+  <arg name="tf_prefix" default="" />
+  <arg if="$(arg sim)" name="pose_estimation" default="false" />
```

Comment:
pose estimation arg is used only in ensenso frames that is used only in sim, so why have this arg in the first place?

Reply:
Fixed in 2a495205b2f992570b23e967f55d40b6991f281d.

Forgot to pipe depth registration arg through to the hardware launch file (which also publishes TFs dependent on pose estimation).